### PR TITLE
Add more logs when pushing build status to github

### PR DESCRIPTION
**Pull request by keyonghan**
Related: https://github.com/flutter/flutter/issues/87512

### CODE REVIEW

File: app_dart/lib/src/request_handlers/push_build_status_to_github.dart [hunk 1]
```diff
@@ -73,7 +73,11 @@ class PushBuildStatusToGithub extends ApiRequestHandler<Body> {
     final Map<LuciBuilder, List<LuciTask>> luciTasks = await luciService.getRecentTasks(builders: postsubmitBuilders);
 
     String status = GithubBuildStatusUpdate.statusSuccess;
-    for (List<LuciTask> tasks in luciTasks.values) {
+    for (LuciBuilder luciBuilder in luciTasks.keys) {
+      final List<LuciTask> tasks = luciTasks[luciBuilder];
```
Comment (jmagman) [inline app_dart/lib/src/request_handlers/push_build_status_to_github.dart:77]: Avoid the value lookup, iterate over the keys and values at the same time.
```suggestion
    luciTasks.forEach((LuciBuilder luciBuilder, List<LuciTask> tasks) {
```

**Review by jmagman — APPROVED** [review]
LGTM with iteration nit.

File: app_dart/lib/src/request_handlers/push_build_status_to_github.dart (hunk 1 above)
Comment (keyonghan) [reply app_dart/lib/src/request_handlers/push_build_status_to_github.dart]: Thanks. Updated.

**Review by jmagman — APPROVED** [review]
LGTM!

**Pull request merged**